bot/extensions/grace_cog.py

```diff
@@ -39,24 +39,30 @@ async def get_contributors_embed(self):
                 value=f"{contributor.contributions} contributions",
                 inline=True
             )
-
         return embed
 
     @hybrid_command(name='change_color', help='Changes the color of the Grace nickname')
-    async def color_command(self, ctx, *, color: str):
-        role = await get_role(ctx.guild.roles, name=self.bot.name)
+    async def color_command(self, ctx, *, color):
+        role = get_role(ctx.guild.roles, name=f"{self.bot.user.name} color")
         if not role:
-            return await ctx.send(f'The bot needs the role {self.bot.name}', ephemeral=True)
+            await ctx.send(f"{self.bot.user.name} color role wasn't found. Creating...", ephemeral=True)
+            role = await ctx.guild.create_role(
+                name=f"{self.bot.user.name} color",
+                permissions=Permissions.advanced()
+            )
+            await (await ctx.guild.fetch_member(self.bot.user.id)).add_roles(role)
 
         await role.edit(colour=Colour.from_str(color))
         await ctx.send(f"Successfully changed Grace color to: {color}", ephemeral=True)
 
     @color_command.error
     async def color_command_error(self, ctx, error):
         if isinstance(error.original.original, ValueError):
-            await ctx.send('Incorrect color format. Acceptable formats are **hex**, **rgb**\n'
-                           'rgb: **rgb(<number>, <number>, <number>)**\n'
-                           'hex: **#<hex>** or **0x<hex>**', ephemeral=True)
+            await ctx.send("Incorrect color format. Acceptable formats are **hex**, **rgb**\n"
+                           "rgb: **rgb(<number>, <number>, <number>)**\n"
+                           "hex: **#<hex>** or **0x<hex>**", ephemeral=True)
+        elif isinstance(error.original.original, Forbidden):
+            await ctx.send("You don't have the permissions to modify this role.", ephemeral=True)
```

Comment:
I am curious about where those strings are sent. Are they sent into an embed, in DM, or in plain text?
If plain text, I would recommend changing it into an embed and sending it via DM.